src/formatting_stack/linters/kondo.clj

```diff
@@ -9,23 +9,31 @@
 
 (def off {:level :off})
 
+(def warn {:level :warning})
+
 (def default-options
   {:cache     true
    :cache-dir formatting-stack.kondo-classpath-cache/cache-dir
-   :linters   {:cond-else            off ;; undesired
-               :missing-docstring    off ;; undesired
-               :unused-binding       off ;; undesired
-               :private-call         off ;; undesired
-               :unresolved-symbol    off ;; can give false positives
-               :unused-symbol        off ;; can give false positives
-               :unused-private-var   off ;; can give false positives
-               :unresolved-var       off ;; already offered by clj
-               :consistent-alias     off ;; already offered by how-to-ns
-               :duplicate-require    off ;; already offered by clean-ns
-               :unused-import        off ;; already offered by clean-ns
-               :unused-namespace     off ;; already offered by clean-ns
-               :unused-referred-var  off ;; already offered by clean-ns
-               :unresolved-namespace off ;; already offered by clean-ns
+   ;; https://github.com/clj-kondo/clj-kondo/blob/v2021.12.19/doc/linters.md
+   :linters   {:docstring-no-summary                  warn ;; disabled by default
```

Comment:
Probably the only controversial one would be `:docstring-no-summary`. I agree with the Clojure Style Guide that complete sentences lead to more comprehensible code.

Feel free to deliberate

Reply:
Didn't add :docstring-no-summary finally, it might be feasible if it adds some extra options